scripts/library_tools.py

```
@@ -0,0 +1,839 @@
+        attr_flag = " [ATTR]" if r.get("attribution_required") else ""
+        print(f"  {r['id']}{attr_flag}")
+        print(f"    name:     {r['name']}")
+        print(f"    category: {r['category']}")
+        print(f"    source:   {r['source']}  ({r['license']})")
+        print(f"    file:     {r['file']}")
+        print()
+
+    return 0
+
+
+def cmd_audit(args: argparse.Namespace) -> int:
```

Ruff (0.15.12)

[warning] Unused function argument: `args`

(ARG001)